docs/dev/allocation.qmd

```diff
@@ -0,0 +1,268 @@
+---
+title: "Allocation"
+---
+# Overview of allocation implementation
+In this document, the allocation workflow is explained. Below is an overview of it.
+```{mermaid}
```

Comment:
This loop is made based on `update_allocation` functions. The looping over priorities is only one of functions called in step `collect_demand` and `allocate`. and the interaction with physical layer is a part of looping priorities. I agree that they are very important steps, maybe to describ it with words below instead of adding them to the graph.